src/Test/intlinear.jl

```diff
@@ -681,6 +681,135 @@ function indicator4_test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+function _semitest(model::MOI.ModelLike, config::TestConfig{T}, int::Bool) where T
+    atol = config.atol
+    rtol = config.rtol
+
+    @test MOIU.supports_default_copy_to(model, #=copy_names=# false)
+    @test MOI.supports(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{T}}())
+    @test MOI.supports(model, MOI.ObjectiveSense())
+    @test MOI.supports_constraint(model, MOI.SingleVariable, MOI.Semicontinuous{T})
+
+    # 2 variables
+    # min  x
+    # st   x >= y
+    #      x ∈ {0.0} U [2.0,3.0]
```

Comment:
This should depend on `int`

Reply:
Something like this?
    # 2 variables
    #
    # If int == false
    # min  x
    # st   x >= y
    #      x ∈ {0.0} U [2.0,3.0]
    #      y = 0.0
    #
    # If int == true
    # min  x
    # st   x >= y
    #      x ∈ {0.0} U {2.0} U {3.0}
    #      y = 0.0

Reply:
Yes but you can do the if only around the semi... constraint. Also, do `int` and `!int`